compute_sdk/tests/unit/test_web_client.py

```diff
@@ -103,9 +104,21 @@ def test_get_amqp_url(client, randomstring):
 
 
 @pytest.mark.parametrize("multi_user", [None, True, False])
-def test_multi_user_post(client, multi_user):
-    responses.post(url="https://api.funcx/v2/endpoints")
-    resp = client.register_endpoint("ep_name", "ep_id", multi_user=multi_user)
+def test_multi_user_post(client: Client, multi_user):
+    responses.post(url="https://api.funcx/v3/endpoints")
+    resp = client.register_endpoint("ep_name", None, multi_user=multi_user)
+    req_body = json.loads(resp._response.request.body)
+    if multi_user:
+        assert req_body["multi_user"] == multi_user
+    else:
+        assert "multi_user" not in req_body
+
+
+@pytest.mark.parametrize("multi_user", [None, True, False])
+def test_multi_user_put(client: Client, multi_user):
+    ep_uuid = uuid.uuid4()
+    responses.put(url=f"https://api.funcx/v3/endpoints/{ep_uuid}")
+    resp = client.register_endpoint("ep_name", ep_uuid, multi_user=multi_user)
```

Comment:
Och; I didn't realize that `responses` had these wrapper methods.  How on earth did I miss that before, and why aren't we using this paradigm more?  Heh.